ci/sal_stack_lwip_ci.py

```diff
@@ -1,5 +1,20 @@
 #!/usr/bin/env python
 
+# PackageLicenseDeclared: Apache-2.0
```

Comment:
So, I'm confused. Is it `PackageLicenseDeclared` or `SPDX-License-Identifier` (https://github.com/ARMmbed/mbed-drivers/blob/master/mbed-drivers/v2/EphemeralBuffer.hpp#L3) now ?